lib/matplotlib/tests/test_colorbar.py

```diff
@@ -712,6 +712,18 @@ def test_colorbar_label():
     assert cbar3.ax.get_xlabel() == 'horizontal cbar'
 
 
+@image_comparison(['colorbar_keeping_xlabel.png'], style='mpl20')
+def test_keeping_xlabel():
+    # github issue #23398 - xlabels being ignored in colorbar axis
+    np.random.seed(seed=19970527)
```

Comment:
Please use the default seed and the "new" way to see numpy: https://matplotlib.org/devdocs/devel/testing.html

```
rng = np.random.default_rng(19680801)
``` 
and then replace `np.random` with `rng`.